server/src/test/java/io/deephaven/server/barrage/BarrageMessageRoundTripTest.java

```diff
@@ -472,13 +500,27 @@ void createNuggetsForTableMaker(final Supplier<Table> makeTable) {
                     RowSetFactory.fromRange(size / 2, size * 3 / 4),
                     subscribedColumns, "floating");
 
+            nuggets.add(new RemoteNugget(makeTable));
+            nuggets.get(nuggets.size() - 1).newClient(
+                    RowSetFactory.fromRange(0, size / 10),
+                    subscribedColumns, true, "footer");
+            nuggets.add(new RemoteNugget(makeTable));
+            nuggets.get(nuggets.size() - 1).newClient(
+                    RowSetFactory.fromRange(size / 2, size * 3L / 4),
+                    subscribedColumns, true, "reverse floating");
+
             final RowSetBuilderSequential swissIndexBuilder = RowSetFactory.builderSequential();
             final long rangeSize = Math.max(1, size / 20);
             for (long nr = 1; nr < 20; nr += 2) {
                 swissIndexBuilder.appendRange(nr * rangeSize, (nr + 1) * rangeSize - 1);
             }
+
+            nuggets.add(new RemoteNugget(makeTable));
+            nuggets.get(nuggets.size() - 1).newClient(swissIndexBuilder.build(), subscribedColumns, "swiss");
```

Comment:
In Java most builders can be built only one time. It is more idiomatic to build and then clone.